util/calcrewards.py

```diff
@@ -4,6 +4,8 @@
 from numpy import log10
 
 from util import cleancase
+from util.query import _symbol
```

Comment:
Given that `_symbol()` is being used outside the original module, please add a unit test for it, and rename to `symbol()`